src/app/dashboard/Missions/ListComponents.jsx

```diff
@@ -1,52 +1,138 @@
 import React from "react";
-import { Box } from "@material-ui/core";
-import { H5 } from "../../component";
+import { Box, Grid } from "@material-ui/core";
 import styled from "styled-components";
 
+import InputLabel from "@material-ui/core/InputLabel";
+import Select from "@material-ui/core/Select";
+import FormControl from "@material-ui/core/FormControl";
+import { makeStyles } from "@material-ui/core/styles";
+import Mission from "../../model/Mission";
+import AccessTimeIcon from "@material-ui/icons/AccessTime";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+
+import _ from "../../utils/lodash";
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+  select: {
+    width: "100px",
+  },
+}));
+
 const SelectedMissionId = styled.div`
   text-decoration: underline;
   cursor: pointer;
   user-select: none;
+  font-weight: bold;
+  padding-left: 11px;
 `;
 
-function MissionName({ onShowDetails, title, type }) {
+const UnasignedStatusAction = ({ missionId }) => {
+  const classes = useStyles();
+  const handleChange = (event) => {
+    event.preventDefault();
+    Mission.update(missionId, {
+      fundedStatus: event.target.value,
+      status: Mission.Status.tentative,
+    });
+  };
```

Comment:
If we're directly updating the mission status via one select change handler, and then the organizer accidentally closes the page, there may be some unintended side effects.

Do we need to consider adding in a confirmation or button to submit a change?

Reply:
yes, I have suggest another flow with modal, but we are going to wait for the next mvp for that

Reply:
Okay cool, just wanted to keep that on the radar